helm/prometheus-rules/templates/platform/atlas/recording-rules/signals.observability-platform.logs.yml

```
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4}}
+  name: signals.observability-platform.logs.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+    - name: signals.observability-platform.logs
+      rules:
+        - record: giantswarm:observability:grafana:unique_users:30d
+          expr: |
+            count by (uname, cluster_id, cluster_type, customer, installation, pipeline, provider, region) (
+                count by (uname, cluster_id, cluster_type, customer, installation, pipeline, provider, region) (
+                    count_over_time(
+                        {container="grafana"}
```

I reckon we don't want to retrieve customer's grafana instances, right?
```suggestion
                        {container="grafana", cluster_type="management_cluster"}
```

---

We do not have this in the logs so we could use the cluster_id and inject it from the values?

---

LGTM